### DISCUSSION TITLE
AD5593 Driver

### DISCUSSION THREAD

**Discussion by alsor62** [micropython/micropython#10864] [Libraries & Drivers] [answered]
Colleagues, did anyone use the ad5593 chip? all functionality - ADC, DAC, GPIO?

**Comment by rkompass** [comment, marked as answer]
Google offered [this](https://github.com/101Robotics/AD5593-MicroPython/blob/main/lib/ad5593.py) .
Which looks like it would work already (try out and/or study the data sheed and check).
If you are interested in re-working this I could offer a collaboration. It's a very interesting chip and a beakout is available on aliexpress.

**Reply by alsor62** [reply]
Yes, I am interested. I want to make a universal I/O board on a single chip

**Reply by rkompass** [reply]
I just ordered two of the AD5593s. I have some code here for different ADCs. In the next days I will put that on Github and add a directory for the AD5593. Then we can communicate there.

**Reply by alsor62** [reply]
Do you plan to add DAC and GPIO output functionality to the AD5593 library?

**Reply by rkompass** [reply]
I think it is already there. And I don't plan to get rid of it :-)
